crates/config/src/madara.rs

```diff
@@ -166,7 +166,7 @@ impl MadaraRunnerConfigSequencer {
 impl Default for MadaraRunnerConfigSequencer {
     fn default() -> Self {
         Self {
-            base_path: Some("madara/data".to_string()),
+            base_path: Some("data/madara".to_string()),
```

Comment:
I like this but Pathfinder also creates a data folder. Can you apply this fix for that too?

Reply:
Done!